src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -1121,6 +1124,14 @@ namespace winrt::TerminalApp::implementation
         }
     }
 
+    // Method Description:
+    // - See Pane::CalcSnappedDimension
+    float TerminalPage::CalcSnappedDimension(const bool widthOrHeight, const float dimension) const
+    {
+        const auto focusedTabIndex = _GetFocusedTabIndex();
+        return _tabs[focusedTabIndex]->CalcSnappedDimension(widthOrHeight, dimension);
```

Comment:
Should this access be via .at() to ensure that it's in range?

Reply:
That's #3181

Reply:
ok ok